webpack.dev.js

```diff
@@ -0,0 +1,17 @@
+const path = require('path');
+/* eslint-disable */
```

Comment:
- [ ] Same case here. As directed in the previous reviewer, we need to reenable linters here and address the issue that was pointed by the linters. Please take care of this.